### PR TITLE
chore(wrappers): add wrapper for plugin-feedback and plugin-feedback-backend (RHIDP 3549)

### PR DESCRIPTION
**Pull request by Fortune-Ndlovu**
## Which issue(s) does this PR fix

[Migrate or support the migration of feedback plugin to backstage/community-plugins](https://issues.redhat.com/browse/RHIDP-3549)

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [x] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: dynamic-plugins/wrappers/backstage-community-plugin-feedback-backend/dist-dynamic/package.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    "short-uuid": "^4.2.2",
+    "yn": "^4.0.0"
+  },
+  "devDependencies": {},
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/janus-idp/backstage-showcase",
```
Comment (nickboldt) [inline dynamic-plugins/wrappers/backstage-community-plugin-feedback-backend/dist-dynamic/package.json:44]: :cry:

File: dynamic-plugins/wrappers/backstage-community-plugin-feedback-backend/dist-dynamic/package.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    "directory": "dynamic-plugins/wrappers/backstage-community-plugin-feedback-backend"
+  },
+  "maintainers": [
+    "@janus-idp/maintainers-showcase"
```
Comment (nickboldt) [inline dynamic-plugins/wrappers/backstage-community-plugin-feedback-backend/dist-dynamic/package.json:48]: :cry:

File: dynamic-plugins/wrappers/backstage-community-plugin-feedback/dist-dynamic/package.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs.js",
+      "default": "./dist/index.cjs.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {},
+  "dependencies": {
+    "@janus-idp/backstage-plugin-feedback": "1.0.1",
```
Comment (nickboldt) [inline dynamic-plugins/wrappers/backstage-community-plugin-feedback/dist-dynamic/package.json:28]: Should this be the MIGRATED plugin in the @community-plugins repo?

Comment (Fortune-Ndlovu) [reply dynamic-plugins/wrappers/backstage-community-plugin-feedback/dist-dynamic/package.json]: Your right, I have updated this

**Comment by nickboldt** [comment]
As far as I can tell the feedback plugin was never in the showcase as of 1.2 or 1.3 CI.

If this is one of the third-party plugins owned by another RH team, and they were not previously included as peer deps in the showcase, then I'm not sure we need this PR to add a dynamic wrapper. 

Can you confirm with @riginoommen @deshmukhmayur @yashoswalyo ?

**Review by riginoommen — APPROVED** [review]

**Comment by nickboldt** [comment]
Closing - we don't plan to bundle any new plugins in RHDH at this time. If you want to be bundled please talk to @christophe-f

**Pull request closed without merging**